tests/Feature/MailExportTest.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace PodPoint\MailExport\Tests\Feature;
+
+use Carbon\Carbon;
+use Illuminate\Mail\Events\MessageSent;
+use Illuminate\Support\Facades\Event;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Facades\Storage;
+use PodPoint\MailExport\Events\MessageStored;
+use PodPoint\MailExport\Tests\Stubs\ExportableByMethodsMailable;
+use PodPoint\MailExport\Tests\Stubs\ExportableByPropertiesMailable;
+use PodPoint\MailExport\Tests\Stubs\ExportableMailable;
+use PodPoint\MailExport\Tests\Stubs\StandardMailable;
+use PodPoint\MailExport\Tests\TestCase;
+
+class MailExportTest extends TestCase
+{
+    /** @test */
+    public function it_can_be_disabled_completely()
+    {
+        config()->set('mail-export', [
+            'enabled' => false,
+            'disk' => 'local',
+            'path' => 'mail-export',
+            'filename' => false,
+        ]);
+
+        Event::fake();
```

Comment:
Potential false positive - this will always pass because faking `Event` here prevents all event handlers from firing as it swaps out the dispatcher, so I don't think the logic ever gets executed.

Reply:
Yeah I had some issue with it. Will take another angle.